scripts/newinstall.sh

```diff
@@ -216,13 +221,27 @@ fi
 
 ##########	Install EUPS
 
+##########	$PYTHON is the Python used to install/run EUPS.
+##########	It can be any Python >= v2.6
+
 if true; then
 	if [[ ! -x "$PYTHON" ]]; then
 		echo -n "Cannot find or execute '$PYTHON'. Please set the PYTHON environment variable or use the -P"
-		echo " option to point to system Python 2 interpreter and rerun."
+		echo " option to point to a functioning Python >= 2.6 interpreter and rerun."
 		exit -1;
 	fi
 
+  PYVEROK=$(python -c 'import sys; print("%i" % (sys.hexversion >= 0x02060000))')
```

Comment:
Shouldn't this one be `$PYTHON` instead of just `python` if it's testing the EUPS one?